### PR TITLE
Derive Clone and Copy for TermType

### PR DESCRIPTION
**Pull request by dvic**

### CODE REVIEW

**Comment by evnu** [comment]
Should we derive `Copy` as well?

**Comment by dvic** [comment]
Good one, I added it!

File: rustler/src/env.rs [hunk 1]
```diff
@@ -310,6 +310,8 @@ impl SavedTerm {
             _ => panic!("can't load SavedTerm into a different environment"),
         }
     }
+
```
[the remark this answers is not in the document]
Comment (dvic) [reply rustler/src/env.rs:313]: I think VSCode did this (all files end with an empty new line I believe)

[the remark this answers is not in the document]
Comment (dvic) [reply rustler/src/env.rs]: Nevermind, it's a different line, fixed it :)

**Comment by evnu** [comment]
Thanks!

**Pull request merged**